web-api/src/business/useCases/user/queueEmailUpdateAssociatedCasesWorker.test.ts

```diff
@@ -9,11 +9,10 @@ import { applicationContext } from '@shared/business/test/createTestApplicationC
 import { mockPetitionerUser } from '@shared/test/mockAuthUsers';
 import { petitionerUser } from '@shared/test/mockUsers';
 import { sleep } from '@shared/tools/helpers';
-import { getCasesByEmailTotal as getCasesByEmailTotalMock } from '@web-api/persistence/postgres/cases/reports/getCasesByEmailTotal';
-
-const getCasesByEmailTotal = getCasesByEmailTotalMock as jest.Mock;
+import { getCasesByEmailTotal as getCasesByEmailTotalMock } from '@web-api/persistence/elasticsearch/getCasesByEmailTotal';
```

Comment:
Is this intentional?

Looks to me like we're rolling something out of postgres and back into elasticsearch?

Reply:
After this was explained in standup today I now understand!